### ISSUE TITLE
[Updating the plugin] UserVoiceShow

### ISSUE THREAD

**Issue by RPPSH** [Vencord/plugin-requests#141]
### Plugin Description

Make UserVoiceShow similar to Voice Activity Can it be in Betterdiscord? 
![preview (1)](https://github.com/Vencord/plugin-requests/assets/103112066/faeca46e-c230-47e7-b6a5-49e950c17147)
![preview](https://github.com/Vencord/plugin-requests/assets/103112066/dfc4978d-5b3c-4f36-9a48-1a8a1c4e39a1)

### Request Agreement

- [X] I read the above and made sure my plugin is possible and allowed
- [X] I checked the [existing plugins](https://vencord.dev/plugins) and made sure my plugin doesn't already exist
- [X] I [searched the existing issues](https://github.com/Vencord/plugin-requests/issues?q=is%3Aissue) and made sure a similar request doesn't already exist

**Comment by EdVraz** [comment]
Why did you attach the same screenshot twice? Can you elaborate on the differences between those 2 plugins and what exactly you would like changed?

**Comment by Soulivan71** [comment]
Yep its great idea, im looking for same thing. I hope the dev gonna update

**Comment by Johannes7k75** [comment]
https://github.com/Vendicated/Vencord/pull/2248
